### PR TITLE
feat: render missed parts for operations and render spec id

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

Render missed parts for operation:
- `operationId`
- `externalDocs`

Render `id` of spec.

ID:
image

Operation:
image

**Related issue(s)**
Part of https://github.com/asyncapi/asyncapi-react/issues/63

### CODE REVIEW

File: playground/src/specs/streetlights.ts [hunk 1]
```diff
@@ -140,7 +146,6 @@ channels:
       streetlightId:
         $ref: '#/components/parameters/streetlightId'
     publish:
-      operationId: turnOff
```
Comment (jonaslagoni) [inline playground/src/specs/streetlights.ts:143]: Why remove the operationId? 🤔

**Review by jonaslagoni — COMMENTED** [review]
Only 1 comment, but, where are the tests? 🧐

File: playground/src/specs/streetlights.ts (hunk 1 above)
Comment (magicmatatjahu) [reply playground/src/specs/streetlights.ts:143]: To check if without `operationId` everything is ok :)

**Comment by magicmatatjahu** [comment]
> Only 1 comment, but, where are the tests? 🧐

We haven't integrated test framework for React.

**Comment by jonaslagoni** [comment]
> We haven't integrated test framework for React.

I see jest is already and a few tests have already been added, shouldn't take you long to test the new output i.e. find operation id and externalDocs and the document id. But of course, features take a little longer to create.

You have to start eventually, so why not this PR? 🤔

**Comment by magicmatatjahu** [comment]
>I see jest is already and a few tests have already been added, shouldn't take you long to test the new output i.e. find operation id and externalDocs and the document id. But of course, features take a little longer to create.

Testing normal TS function isn't this same as testing React, it isn't testing html as string and make some regex. Also we will rewrite component again in near future, when we will have a designer and therefore I don't want to test every part of a component.

**Comment by jonaslagoni** [comment]
> Testing normal TS function isn't this same as testing React, it doesn't like testing html as string and make some regex. Also we will rewrite component again in near future, when we will have a designer and therefore I don't want to test every part of a component.

Okay 🙂

**Review by jonaslagoni — APPROVED** [review]

**Pull request merged**